package.json

```diff
@@ -52,6 +52,8 @@
     "test-requirejs": "node tasks/test_requirejs.js",
     "test-plain-obj": "node tasks/test_plain_obj.js",
     "test": "npm run test-jasmine -- --nowatch && npm run test-bundle && npm run test-image && npm run test-export && npm run test-syntax && npm run lint",
+    "mathjax3": "node devtools/test_dashboard/server.js --mathjax3",
+    "mathjax3chtml": "node devtools/test_dashboard/server.js --mathjax3chtml",
```

Comment:
🌴  slightly cleaner (also for `strict`): `npm run start -- --mathjax3`

Reply:
Thanks for the suggestion. But let's skip this for now.